Demo/Demo/Models/PaymentTokenResponse.swift

```diff
@@ -14,7 +14,8 @@ struct Customer: Codable, Equatable {
 
 struct PaymentSource: Decodable, Equatable {
     
-    let card: BasicCard
+    var card: BasicCard?
+    var paypal: BasicPayPal?
 }
 
 struct BasicCard: Decodable, Equatable {
```

Comment:
What does BasicCard and BasicPayPal mean?

Reply:
meaning they contain very minimal results like last four digits and email.

Reply:
2cents: On Android we went with `CardPaymentSource` and `PayPalPaymentSource`. JSON encoding of this request is weird on Android too.